blog/Dockerfile.nginx

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM {{ hail_ubuntu_image.image }}
```

Comment:
Does this image have nginx in it? Just trying to think of all the components that are not tested.

Reply:
The hail_ubuntu_image does not have nginx. The blog image does have nginx (its installed on line 4 of this file).